platooning_control_ihp/src/platoon_control_ihp.cpp

```diff
@@ -49,25 +49,19 @@ namespace platoon_control_ihp
         pnh_->param<double>("lowpassGain", config.lowpassGain, config.lowpassGain);
         pnh_->param<double>("lookaheadRatio", config.lookaheadRatio, config.lookaheadRatio);
         pnh_->param<double>("minLookaheadDist", config.minLookaheadDist, config.minLookaheadDist);
-
-        // Global params (from vehicle config)
```

Comment:
Could you please restore this comment, as it helps the reader to understand that the remaining params are not from the local config file.